util/ranger/testdata/ranger_suite_out.json

```diff
@@ -123,7 +123,7 @@
       {
         "SQL": "select * from t where a = 1 and (b = 1 or b = 2) and b = 3 and c > 1;",
         "Plan": [
-          "TableDual_5 0.00 root  rows:0"
+          "TableDual_5 8000.00 root  rows:0"
```

Comment:
The  `estRows` does not look as expected.

Reply:
when 'stats' is 'pseudo'，this value doesn't make much sense.